### DISCUSSION TITLE
ds<char><char> with O-Pending mode

### DISCUSSION THREAD

**Discussion by GustafXTT** [ggandor/leap.nvim#189] [General]
I heard that some people said leap.nvim doesn't support deleting until certain labels by using `ds<char>char>`, and I am wondering if it is supported by leap.nvim officially.
I tested a bit and found leap.nvim actually works, you need to wait for 1 second after typing the first `d` to wait for the nvim to enter the Operator-Pending mode, then `s<char><char>` to delete until certain labels.

**Comment by ggandor** [comment]
> you need to wait for 1 second after typing the first d to wait for the nvim to enter the Operator-Pending mode

You don't have to wait, type `ds<char><char>` in one go, it will work.
